src/instructlab/model/train.py

```diff
@@ -304,6 +305,28 @@ def train(
     Takes synthetic data generated locally with `ilab data generate` and the previous model and learns a new model using the MLX API.
     On success, writes newly learned model to {model_dir}/mlx_model, which is where `chatmlx` will look for a model.
     """
+
+    if not legacy and os.path.isdir(data_path):
+        # need to get msgs jsonl, this is a hack but will work for now
+        pattern = os.path.join(data_path, f'*{"messages"}*')
+
+        # Find all files matching the pattern
+        files = glob.glob(pattern)
+
+        # If no files are found, error
+        if not files:
+            ctx.fail(
```

Comment:
AFAIK `ctx.fail()` will print the help message for the command. Do we want that?

Reply:
It's probably acceptable, but I also agree that posting a Usage message may be confusing, since it suggests that the format of the CLI arguments is invalid. In reality, it is.